TKGame/Components/Concrete/Enemy_GraphicsComponent.cs

```diff
@@ -13,13 +13,11 @@
 
 namespace TKGame.Components.Concrete
 {
-    internal class Enemy_GraphicsComponent : GraphicsComponent
+    class Enemy_GraphicsComponent : GraphicsComponent
     {
         void GraphicsComponent.Update(Entity entity, SpriteBatch spriteBatch)
         {
             Player player = Player.Instance;
-            entity.Velocity.X = 1;
-            entity.Velocity.Y = 1;
 
             //spriteBatch.Begin(SpriteSortMode.Deferred, BlendState.NonPremultiplied);
             spriteBatch.Draw(entity.entityTexture,
```

Comment:
You should draw the sprite on this frame after all graphics have been updated. aka just put this below the Orientation stuff

Reply:
Currently it is drawing according to the last frame